affinity/cpp-20/d0796r1.md

```diff
@@ -165,94 +206,163 @@ There are some additional challenges which we have been investigating but are no
 
 ## Class `execution_resource`
 
-The `execution_resource` class provides an abstraction over a system's hardware capable of memory allocation, execution of light weight exeution agents or both.
+The `execution_resource` class provides an abstraction over a system's hardware capable of memory allocation, execution of light weight exeution agents or both. An `execution_resource` can represent further `execution_resource`s, these `execution_resource`s are said to be *members of* this `execution_resource`.
 
-### `execution_resource` constructors
+> [*Note:* The `execution_resource` is required to be implemented such that the underlying software abstraction is initialised on when the `execution_resource` is constructed, maintained through reference counting and cleaned up on destruction of the final reference. *--end note*]
```

Comment:
on when -> when
why the reference counting?

Reply:
Fixed.
See earlier comment on reference counting.